src/utils.js

```diff
@@ -0,0 +1,6 @@
+/* @flow */
+
+// Node-style callback
+// FIXME: Could not find a way to define a Callback<T> type with multiple
+//        signatures.
+export type Callback = (?Error, any) => void;
```

Comment:
Oh, ok, makes sense.
I would prefer to wait until I encounter the case, so I can make sure the type modification actually works on existing code.